rampwf/utils/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .command_line import ramp_test_submission
```

Comment:
I don't think it is needed to import this here? (for the console script, you access it from utils.command_line no?)